integration-tests/api-gateway/endpoints.ts

```diff
@@ -1,124 +1,87 @@
-import { getSSMParameter } from "../resources/ssm-param-helper";
-import {
-  Payload,
-  getJarAuthorizationPayload,
-} from "./crypto/create-jar-request-payload";
-import {
-  getClaimSet,
-  CLIENT_ID,
-  CLIENT_URL,
-  environment,
-} from "./env-variables";
+import { environment } from "./env-variables";
 import { stackOutputs } from "../resources/cloudformation-helper";
-
-let publicEncryptionKeyBase64: string;
-let privateSigningKey: any;
-let privateAPI: string;
-let preOutput: Partial<{
-  PrivateApiGatewayId: string;
-}>;
-
-type Name = {
-  name: {
-    nameParts: {
-      type: string;
-      value: string;
-    }[];
-  }[];
+import { signedFetch } from "../resources/fetch";
+
+export type JWTClaimsSet = {
+  iss: string;
+  sub: string;
+  aud: string;
+  iat: number;
+  exp: number;
+  nbf: number;
+  response_type: string;
+  client_id: string;
+  redirect_uri: string;
+  state: string;
+  govuk_signin_journey_id: string;
+  shared_claims?: undefined;
+  evidence_requested?: undefined;
+  context?: string;
 };
 
-export const createPayload = async (sharedClaimsUpdate?: Name) => {
-  publicEncryptionKeyBase64 =
-    (await getSSMParameter(
-      "/check-hmrc-cri-api/test/publicEncryptionKeyBase64"
-    )) || "";
-  privateSigningKey = JSON.parse(
-    (await getSSMParameter("/check-hmrc-cri-api/test/privateSigningKey")) || ""
-  );
-  preOutput = await stackOutputs(process.env.STACK_NAME);
-  privateAPI = `${preOutput.PrivateApiGatewayId}`;
-  const correctClaimSet = await getClaimSet();
-  const updateClaimset = {
-    ...correctClaimSet,
-    ...sharedClaimsUpdate,
-    name: sharedClaimsUpdate?.name || correctClaimSet.shared_claims.name,
-  };
-  const audience = correctClaimSet.aud;
-  const payload = {
-    clientId: CLIENT_ID,
-    audience,
-    authorizationEndpoint: `${audience}/oauth2/authorize`,
-    redirectUrl: `${CLIENT_URL}/callback`,
-    publicEncryptionKeyBase64: publicEncryptionKeyBase64,
-    privateSigningKey: privateSigningKey,
-    issuer: CLIENT_URL,
-    claimSet: updateClaimset,
-  } as unknown as Payload;
-  const ipvCoreAuthorizationUrl = await getJarAuthorizationPayload(payload);
-  return ipvCoreAuthorizationUrl;
-};
-
-export const createSession = async (): Promise<Response> => {
-  const ipvCoreAuthorizationUrl = await createPayload();
-  const sessionApiUrl = `https://${privateAPI}.execute-api.eu-west-2.amazonaws.com/${environment}/session`;
-  const sessionResponse = await fetch(sessionApiUrl, {
+export const getJarAuthorization = async (
+  clientId?: string,
+  aud?: string,
+  iss?: string,
+  claimsOverride?: unknown,
+  evidence_requested?: unknown
+) => {
+  const { TestHarnessExecuteUrl: testHarnessExecuteUrl } =
+    await stackOutputs("test-resources");
```

Comment:
Should this be an env var so we can easily change the test-resources stack used?